sw/device/lib/testing/test_framework/BUILD

```diff
@@ -124,9 +121,12 @@ cc_library(
         "-Wl,--start-group",
         "$(location :freertos_port)",
         "$(location //sw/device/lib:irq)",
-        "$(location //sw/vendor/freertos_freertos_kernel:kernel)",
+        "$(location //third_party/freertos)",
         "-Wl,--end-group",
     ],
+    # Temporary hack to allow us to #include the version in //third_party.
```

Comment:
Would you mind filing a related issue (with matching TODO here) for this? So we don't lose track of this.